File: application/app/src/main/java/com/yahyeet/boardbook/activity/home/matchfeed/IMatchfeedFragment.java
package com.yahyeet.boardbook.activity.home.matchfeed;

/**
 * Abstract the match feed
 */
public interface IMatchfeedFragment {


	/**
	 * Initiates recyclerView of matches with a adapter
	 */
	void enableMatchFeed();

	/**
	 * Orders adapter of matches to repopulate itself
	 */
	void repopulateMatchFeed();

}

File: application/app/src/main/java/com/yahyeet/boardbook/activity/home/matchfeed/IMatchfeedFragment.kt
package com.yahyeet.boardbook.activity.home.matchfeed

interface IMatchfeedFragment {


    /**
     * Initiates recyclerView of matches with a adapter
     */
    fun enableMatchFeed()

    /**
     * Orders adapter of matches to repopulate itself
     */
    fun repopulateMatchFeed()

}
